### PR TITLE
wip: bb blog post

### PR DESCRIPTION
**Pull request by jacobdadams**

### CODE REVIEW

File: _posts/2023-03-03-broadband-fabric-data-challenge.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  - data
+  - broadband
+---
+
+_From the Teamwork Makes the Dream Work department_
+{: .flex .flex--center}
+
+Among the many lessons the Covid-19 pandemic taught us, one that was hammered home almost immediately was the importance of high-speed broadband internet access at home. As work-from-home and simultaneous video calls became commonplace, we discovered that the asymmetrical, download-focused broadband of yesteryear just didn't cut it anymore. We also discovered that figuring out what broadband was available in our neighborhood was a more difficult task than it had to be.
+
+Unfortunately, this isn't new—the state and federal governments have been struggling with poor data on broadband availability for years. While the FCC required providers to report coverage, the reliance on census block geometries made it impossible to get an accurate and fine-grained picture. Without accurate data, governments have struggled to direct resources and policy to areas that are lacking coverage or the competition necessary for free market economics to do their thing. Ultimately, consumers have often been the ones to suffer slow speeds and high prices.
```
Comment (steveoh) [inline _posts/2023-03-03-broadband-fabric-data-challenge.md:19]: **thought**: these fancy dash options are available to you if you so please.
image

File: _posts/2023-03-03-broadband-fabric-data-challenge.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+We can all rejoice that improvement is coming. In 2019, the FCC [voted](https://www.fcc.gov/document/fcc-improves-broadband-mapping) to collect more granular data and hopefully improve some of the [long-standing problems](https://arstechnica.com/tech-policy/2019/08/the-fccs-horrible-broadband-mapping-system-is-finally-getting-an-upgrade/) with the existing system. As part of these efforts, the FCC invited state and local governments to help with the process to ensure that the maps are more accurate.
+
+UGRC worked with the [Utah Broadband Center](https://business.utah.gov/broadband/) and used our existing relationships with county GIS departments to respond to this invitation. **This cooperation and collaboration enabled Utah to have, by far, the largest number of corrections accepted in the nation.** This would not have been possible without the hard work (on a tight deadline) of the counties who were able to participate. We express a heart-felt **Thank You!** to Box Elder, Weber, Tooele, Salt Lake, Utah, Summit, and Wayne counties for participating on such short notice.
+
+<!-- image of challenge results -->
+
+## Every Program Needs Some Acronyms
+{: .flex .flex--right}
```
Comment (steveoh) [inline _posts/2023-03-03-broadband-fabric-data-challenge.md:28]: these can be simplified to .text-left. I don't think flexbox needs to be used.

**Review by stdavis — APPROVED** [review]
Awesome work, as usual.

**Comment by stdavis** [comment]
I'll leave it up to you when you want this merged...

**Comment by jacobdadams** [comment]
Shoot, my update nixed your approved review. Once you approve it will auto-merge.

**Review by stdavis — APPROVED** [review]

**Pull request merged**